parser-typechecker/src/Unison/Codebase/Editor/Command.hs

```diff
@@ -61,6 +61,9 @@ data Command m i v a where
   -- Presents some output to the user
   Notify :: Output v -> Command m i v ()
 
+  -- Presents output to the user (non-modal/paged)
+  Notify' :: Output v -> Command m i v ()
```

Comment:
Would you mind renaming this to maybe `NotifyNonpaged`?  I think we are already suffering too many `'`-suffixed definitions. 😅

Reply:
Fixed!